docs/platform/release-notes/tizen-6-5-m2.md

```diff
@@ -0,0 +1,400 @@
+# Tizen 6.5 Public M2
+
+Release date: Oct. 31, 2021
+
+

```

Comment:
It will be added by mijin85cho later.
Please refer to the following links.
https://github.com/Samsung/tizen-docs/pull/1439#discussion_r631276601
https://github.com/Samsung/tizen-docs/pull/1201#discussion_r470166198
https://github.com/Samsung/tizen-docs/pull/1201#discussion_r470166601

Reply:
Thanks for confirming